committee-code-of-conduct/README.md

```diff
@@ -15,10 +15,10 @@ The [charter](charter.md) defines the scope and governance of the Code of Conduc
 ## Members
 
 * Ana Margarita Medina (**[@AnaMMedina21](https://github.com/AnaMMedina21)**), Upbound
-* Danielle Lancashire (**[@endocrimes](https://github.com/endocrimes)**), Independent
-* Hilliary Lipsig (**[@hlipsig](https://github.com/hlipsig)**), Red Hat
+* Divya Mohan (**[@divya-mohan0209](https://github.com/divya-mohan0209)**), SUSE
+* Danielle Lancashire (**[@endocrimes](https://github.com/endocrimes)**), Fermyon
 * Jeremy Rickard (**[@jeremyrickard](https://github.com/jeremyrickard)**), Microsoft
-* Xander Grzywinski (**[@salaxander](https://github.com/salaxander)**), Independent
+* Sean McGinnis (**[@stmcginnis](https://github.com/stmcginnis)**), Lambda Labs
```

Comment:
Minor thing only if there is reason to update - my company has done a small rebranding and I'm told we should not be referred to as either Lambda, Inc. or just Lambda. 🤷‍♂️

Reply:
Updated!